ui/pages/login.tsx

```diff
@@ -235,7 +238,9 @@ const Login: NextPage = () => {
       ui: {
         ...flow.ui,
         nodes: flow.ui.nodes.filter(({ group }) => {
-          return useBackupCode ? group !== "totp" : group !== "lookup_secret";
+          return useBackupCode
+            ? group !== "totp" && group !== "webauthn"
```

Comment:
This might have side effects in other flows. E2e tests are fine, though they don't cover all possible flows and screens.